featuretools/variable_types/variable.py

```diff
@@ -121,97 +125,196 @@ def interesting_values(self, values):
+                 entity,
+                 name=None,
+                 true_values=None,
+                 false_values=None):
+        default = [1, True, "true", "True", "yes", "t", "T"]
+        self.true_values = true_values or default
+        default = [0, False, "false", "False", "no", "f", "F"]
+        self.false_values = false_values or default
+        super(Boolean, self).__init__(id, entity, name=name)
+
+    def create_data_description(self):
+        description = super(Boolean, self).create_data_description()
+        description['properties'].update({
+            'true_values': self.true_values,
+            'false_values': self.false_values
+        })
+        return description
+
 
 class Categorical(Discrete):
-    """Represents variables that can take an unordered discrete values"""
-    _dtype_repr = "categorical"
+    """Represents variables that can take an unordered discrete values
+
+    Args:
+        categories (list) : List of categories. If left blank, inferred from data.
+    """
+    type_string = "categorical"
+
+    def __init__(self, id, entity, name=None, categories=None):
+        self.categories = None or []
+        super(Categorical, self).__init__(id, entity, name=name)
+
+    def create_data_description(self):
+        description = super(Categorical, self).create_data_description()
+        description['properties'].update({'categories': self.categories})
+        return description
 
 
 class Id(Categorical):
     """Represents variables that identify another entity"""
-    _dtype_repr = "id"
+    type_string = "id"
     _default_pandas_dtype = int
 
 
 class Ordinal(Discrete):
     """Represents variables that take on an ordered discrete value"""
-    _dtype_repr = "ordinal"
+    type_string = "ordinal"
     _default_pandas_dtype = int
 
 
 class Numeric(Variable):
     """Represents variables that contain numeric values
 
+    Args:
+        range (list, optional) : List of start and end. Can use inf and -inf to represent infinity. Unconstrained if not specified.
+        start_inclusive (bool, optional) : Whether or not range includes the start value.
+        end_inclusive (bool, optional) : Whether or not range includes the end value
+
     Attributes:
         max (float)
         min (float)
         std (float)
         mean (float)
     """
-    _dtype_repr = "numeric"
+    type_string = "numeric"
     _default_pandas_dtype = float
 
+    def __init__(self,
+                 id,
+                 entity,
+                 name=None,
+                 range=None,
+                 start_inclusive=True,
+                 end_inclusive=False):
+        self.range = None or []
+        self.start_inclusive = start_inclusive
+        self.end_inclusive = end_inclusive
+        super(Numeric, self).__init__(id, entity, name=name)
+
+    def create_data_description(self):
+        description = super(Numeric, self).create_data_description()
+        description['properties'].update({
```

Comment:
this is should to go into the `type` field not `properties`

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/361/commits/bbfb55bf48a0d154860e80a4c53ea92873b4cc28)